tools/interactive/interactivetool_anylabeling.xml

```diff
@@ -17,18 +17,18 @@
 	 chown -R 1000:1000 ./home &&
 
          #for $i,$img in enumerate($input_images):
-            ln -s '$img' './home/input_images/${img.element_identifier}' &&
+	    ln -s '$img' './home/input_images/${img.element_identifier}.${img.ext}' &&
```

Comment:
When the name now looks like foobar.jpeg.jpeg is that a problem? If so you might what to check that conditionally.